bq-connector/docai_bq_connector/helper/__init__.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2022 Google LLC
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def find(f, seq):
+    """https://tomayko.com/blog/2004/cleanest-python-find-in-list-function"""
+    """Return first item in sequence where f(item) == True."""
+    for item in seq:
+        if f(item):
+            return item
+
+
+def get_bool_value(raw_value) -> bool:
+    if raw_value in (1, "True" "CHECKED"):
+        return True
+    return False
+
+
+def clean_number(raw_value):
+    if "$" in raw_value or "," in raw_value or "." in raw_value:
+        raw_value = raw_value.replace("$", "")
+        raw_value = raw_value.replace(",", "")
+    return raw_value
```

Comment:
Is there a need for these functions when you can get [Normalized Values][1] from Document AI Entity Extraction processors? I could see this being needed for the form parser, but I'm not sure if that's part of the intended usage.

NOTE: the list of supported fields in the documentation is likely not complete.

[1]: https://cloud.google.com/document-ai/docs/ekg-enrichment

Reply:
@holtskinner I checked the resultant normalized_values, and not a single instance has values outside of the default 0's and 'False'. I think this is because of the parser.